### PR TITLE
DEV-26421: Created V2 for WhoAmI, added account ID parameter

### PR DESCRIPTION
**Pull request by YajB**
# What changed

https://deploy-preview-1131--logz-docs.netlify.app/api/#tag/Who-am-I

- updated API to v2, with the addition of a parameter to retrieve the Account ID. 

- improved descriptions

Relates to [DEV-26421](https://logzio.atlassian.net/browse/DEV-26421)

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

**Review by alexgnatyuk — CHANGES_REQUESTED** [review]
@YajB Please hold on with this one for a little bit. We got an internal request to change the URL again, it is in progress now. I'll let you know once it is done - shouldn't take to long.

File: _source/api/logzio-public-api.yml [hunk 1]
```diff
@@ -3895,13 +3896,18 @@ definitions:
...
     properties:
+      accountID:
+        type: integer
+        format: int32
+        description: Account ID 
+        example: 123456
       accountName:
         type: string
         description: Name of the account
-        example: Jean Valjean
```
Comment (AmiStrn) [inline _source/api/logzio-public-api.yml:3904]: this would of course require the id in the example to be 24601 😄

Comment (YajB) [reply _source/api/logzio-public-api.yml:3904]: Since it's an account name - why not "Deployment Staging account"

Comment (AmiStrn) [reply _source/api/logzio-public-api.yml:3904]: i dont mind what the name is in the end, but if we are going for Jean Valjean then the id has got to be 24601 😄

Comment (AmiStrn) [reply _source/api/logzio-public-api.yml:3904]: https://www.youtube.com/watch?v=nXhofhspBd4

Comment (YajB) [reply _source/api/logzio-public-api.yml:3904]: I completely LURVE your suggestion!
Now that I get it.

**Review by shishkinnico — COMMENTED** [review]
Agree - this is straightforward.

**Review by shishkinnico — APPROVED** [review]

**Pull request merged**